docs/getting-started.md

```diff
@@ -0,0 +1,91 @@
+
+```sh
+code hello-world
+```
+
+Press F5 or open the debug view and start the available debug configuration to launch the extension in a new `Extension Development Host` window. Open a folder and create a file with your chosen file extension (`.hello` is the default). The `hello-world` language accepts two kinds of entities: The `Person` and `Hello` entity. Here's a quick example on how to use them both:
+
+```
+person Alice
+Hello Alice!
+
+person Bob
+Hello Bob!
+```
+
+And here's the Langium grammar that parses this file:
+
+```
+grammar HelloWorld hidden(WS)
+
+terminal WS: /\s+/;
+terminal ID: /[_a-zA-Z][\w_]*/;
+
+Model: (persons+=Person | greetings+=Greeting)*;
+
+Person:
+    'person' name=ID;
+
+Greeting:
+    'Hello' person=[Person] '!';
+```
+
+## Explaining the grammar
+
+Lets go through this one by one:
+
+```
+grammar HelloWorld hidden(WS)
+```
+
+Before we tell Langium anything about our grammar contents, we first need to give it a name - in this case it's `HelloWorld`. The `langium-cli` will pick this up to prefix any generated classes with this name. Additionally we define here which terminals we want to hide from our parser. This will make more sense once we go to the next step.
+
+```
+terminal WS: /\s+/;
+terminal ID: /[_a-zA-Z][\w]*/;
+```
+
+Here we define our two needed terminals for this grammar: The whitespace `WS` and identifier `ID` terminals. Terminals parse a part of our input text by matching it against their regular expression. The `WS` terminal parses any whitespace characters with the regex `/\s+/`. This allows us consume whitespaces in our input text. As the terminal is referenced by the grammar as a hidden terminal, the parser will parse any whitespace and discard the results. That way, we don't have to care about how many whitespaces a user uses in their input text. Secondly, we define our `ID` terminal. As per its regex, it parses any string that starts with an underscore or letter and continues with any amount of characters that match the `\w` regex token. It will match `Alice`, `_alice`, or `_al1c3` but not `4lice` or `#alice`. Langium is using the JS regex dialect for terminal definitions.
+
+```
+Model: (persons+=Person | greetings+=Greeting)*;
+```
+
+The `Model` parser rule is the entry point to our grammar. Input text will always go through that parser rule. It defines a repeating group of alternatives. The alternative in question is `persons+=Person | greetings+=Greeting`. This will always try to parse either a `Person` or a `Greeting` and add it to the list of `persons` or `greetings` in the `Model` object. Since the alternative is wrapped in a repeating group `*`, the parser will continue until all input has been parsed. For more information on how the objects created by your grammar are constructed, visit the [abstract syntax tree documentation](./features/ast.md). 
```

Comment:
Maybe omit this reference to AST documentation because it's already at the end? A forward-reference in the middle can be distracting.